www/front_src/src/helpers/dynamicImport.js

```diff
@@ -17,18 +19,22 @@ export function dynamicImport(basename, parameters) {
       }
 
       // check external component in memory to avoid to reimport it
-      const vector = "$centreonExternalModule$" + parameters.js.replace(/(^\.?\/)|(\.js)/g, '').replace(/\//g, '$');
+      const vector = '$centreonExternalModule$' + parameters.js.replace(/(^\.?\/)|(\.js)/g, '').replace(/\//g, '$');
       if (typeof(window[vector]) === "object") {
         return resolve(window[vector]);
       } else {
         const module = await(window.System.import(basename + parameters.js));
-        window[vector] = module;
+        if (module.default && typeof(module.default) === 'object') { // named umd export
+          window[vector] = module.default;
+        } else { // unnamed umd export or systemjs export
+          window[vector] = module;
+        }
         return resolve(window[vector]);
       }
     } catch (error) {
-      return reject(error);
+      console.error(error);
```

Comment:
please be more specific, ie:  "dynamicImport error : " + error

Reply:
error is an object, not a string.
I will print the stack trace, so no need to display more information